_examples/golang-sse/webapp/src/rpc.gen.ts

```diff
@@ -486,8 +485,13 @@ export interface WebrpcStreamOptions<T> extends WebrpcOptions {
   onOpen?: () => void;
   onClose?: () => void;
 }
+
 export interface WebrpcOptions {
   headers?: HeadersInit;
-  signal?: AbortSignal;
+  controller?: AbortController;
```

Comment:
I renamed from `signal` to `controller` here. Let me know if you guys think this is fine or if it breaks any functionality / uses from before?